course/page/base.py

```diff
@@ -139,15 +139,57 @@ def markup_to_html(
 
 # {{{ answer feedback type
 
+
+class InvalidFeedbackPointsError(ValueError):
+    pass
+
+
+CORRECTNESS_ATOL = 1e-5
+
+
+def get_close_value(correctness):
```

Comment:
I'd move all this logic into one function, `validate_point_count`

Reply:
Good point!